### PR TITLE
Upgrade to Polymer 2.x hybrid

### LINKED ISSUE #2: Upgrade to Polymer 2.x support
Update this web component to Polymer 2.x support.

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**
Fix #2.

### CODE REVIEW

**Comment by FluorescentHallucinogen** [comment]
@abdonrd PTAL.

**Review by abdonrd — APPROVED** [review]
Thanks @FluorescentHallucinogen!

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-bower_components/
+bower_components*/
+bower-*.json
```
Comment (FluorescentHallucinogen) [inline .gitignore:2]: @abdonrd What is it for?

Comment (abdonrd) [reply .gitignore:2]: Try `polymer install --variants`. :)

**Comment by FluorescentHallucinogen** [comment]
@abdonrd Could you please create new Git tag and GitHub release. :wink:

**Pull request merged**